Header header
time time
uint8 robot_id